packages/visx-xychart/test/components/LineSeries.test.tsx

```diff
@@ -15,37 +14,37 @@ describe('<LineSeries />', () => {
   });
 
   it('should render a LinePath', () => {
-    const wrapper = mount(
+    const { container } = render(
       <DataContext.Provider value={getDataContext(series)}>
         <svg>
           <LineSeries dataKey={series.key} {...series} />
         </svg>
       </DataContext.Provider>,
     );
     // @ts-ignore produces a union type that is too complex to represent.ts(2590)
```

Comment:
I saw this error went away in one of the tests above, wonder if the same is true here? I'd like to do a big sweep an swap all `@ts-ignore` => `@ts-expect-error` soon so that we don't have lingering `ts-ignore`s around

Reply:
good catch, will fix it